osversion.go

```diff
@@ -6,6 +6,7 @@ type OSVersion struct {
 	IDLike  string
 	Name    string
 	Version string
+	AdditionalFlags map[string]string
```

Comment:
Maybe "Extra" or "ExtraFields". Flags seems wrong.

Reply:
agree. I will get that updated

Reply:
I did follow your approach, but then switched to accessors. This needs your opinion.

Reply:
went back to your approach as discussed. This should be resolved now.